llvm/include/llvm/Support/MathExtras.h

```diff
@@ -355,9 +372,14 @@ constexpr inline uint64_t MinAlign(uint64_t A, uint64_t B) {
   return (A | B) & (1 + ~(A | B));
 }
 
+/// Fallback when arguments aren't integral.
```

Comment:
Why do you need fallbacks like this? What kind of "non-integral" arguments are used?

Reply:
Enums, for instance.